cylc/flow/scheduler.py

```diff
@@ -364,7 +360,6 @@ async def initialise(self):
             self.suite, context=self.zmq_context, barrier=self.barrier)
 
         self.proc_pool = SubProcPool()
-        self.state_summary_mgr = StateSummaryMgr()
```

Comment:
Woohoo!

Reply:
There might be some `TaskProxy.summary` to get rid of too.. But I'll leave that for when the data store moves to event driven updates.